### PR TITLE
feat: add `string/base/last`

### LINKED ISSUE #1696: [RFC]: add `@stdlib/string/base/last`
### Description

This RFC proposes adding the `string/base/last` package which will return the last `n` UTF-16 code units of a string.

Signature:
`last( string, n )`

Eg.
```
var s = first( 'hello', 1 );
// returns 'o'
```

Package: `@stdlib/string/base/last`
Alias: last

This will help with development of `@stdlib/string/last` issue #854

### Related Issues

Related issues #854  

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by adityacodes30**
Resolves #1696 

## Description

> What is the purpose of this pull request?

This pull request adds the package `@stdlib/string/base/last` 

This pull request:

-   resolves #1696 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by adityacodes30** [comment]
tests pass - this pr is ready for review @kgryte @Planeshifter

File: lib/node_modules/@stdlib/string/base/last/lib/main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* var out = last( 'JavaScript', 6 );
+* // returns 'Script'
+*
+* @example
+* var out = last( 'New', 6 );
+* // returns 'New'
+*/
+function last( str, n ) {
+	var len;
+	len = str.length;
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/string/base/last/lib/main.js]: While we do place the `var` declarations at the top to reflect the hoisting, when the assignment immediately follows we can collapse into one expression:

```suggestion
	var len = str.length;
```

File: lib/node_modules/@stdlib/string/base/last/test/test.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	s = last( 'new', 3 );
+	t.strictEqual( s, 'new', 'returns expected value' );
+
+	s = last( '<--->', 2 );
+	t.strictEqual( s, '->', 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns whole string if `n` is greater than length of string', function test( t ) {
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/string/base/last/test/test.js]:
```suggestion
tape( 'the function returns the original string if `n` is greater than the length of the string', function test( t ) {
```

**Review by Planeshifter — APPROVED** [review]
Looks good, thank you @adityacodes30!

**Pull request merged**